nemo/collections/llm/recipes/llama3_70b.py

```diff
@@ -93,6 +95,26 @@ def pretrain_recipe(
     )
 
 
+def pretrain_recipe_performance(
+    name: str, ckpt_dir: str, num_nodes: int, num_gpus_per_node: int, fn: Callable = pretrain
+) -> Partial:
```

Comment:
Doc-string that explains this is a performant version and is prone to some errors. 

link the fallback to pretrain_recipe() for safer.